README.md

```diff
@@ -387,6 +384,15 @@ let results2 = statement.execute();
 
 You only pay the price of parsing the query once, and each subsequent execution should be faster.
 
+# Raw execution
+
+If you don't care about the keys you can use a simplified execution that will return an array of results.
+
+```ts
+let result = await db.executeRaw('SELECT * FROM Users;');
```

Comment:
Since it's async shouldn't it be `executeAsyncRaw`? Is there also a sync version?

Reply:
Got lazy, will do later

Reply:
Can I try to contribute this feature?

Reply:
This is already merged, sorry buddy

Reply:
There is still no sync version is not it?
![image](https://github.com/OP-Engineering/op-sqlite/assets/57757211/cc193763-d813-4388-83f8-464165fbcc8d)

Reply:
I actually would like to remove all sync versions because they make the codebase larger than they need to be. Except for the most simple of use cases async should always be the go to.

Reply:
sync can be very useful in some cases - would be great to keep them

Reply:
Yeah, i won't remove any of the stuff existing ones, but won't add more